### ISSUE TITLE
www.wikidata.org

### ISSUE THREAD

**Issue by MacSquirrel-Jedi** [gnosygnu/xowa#238]
hello gnosygnu,

xowa_app_windows_64_v4.5.21.1808
www.wikidata.org (Import online)
1) Import was successful, interlinks works. But there was 2 attentions at the end of import

![xowa_wikidata_online_import 1](https://user-images.githubusercontent.com/18243759/44414130-4f954100-a56d-11e8-9ead-2c023688c591.jpg)

![xowa_wikidata_online_import 2](https://user-images.githubusercontent.com/18243759/44414173-6c317900-a56d-11e8-9238-833dfe79f887.jpg)

but files below were generated
www.wikidata.org-xtn.search.core.xowa
www.wikidata.org-xtn.search.link-title-ns.000-db.001.xowa
www.wikidata.org-xtn.search.link-title-ns.999-db.001.xowa

Is it just wrong attention or should i upgrade category (don't know reason) ? It takes over 1 day to generate www.wikidata.org

2) Please, what mean Page ranks? 
When I'm searching for example Bible in XOWA, first is "United Bible Societies". When i use internet, first "Bible". So when i set wiki-date-pagelinks.sql.gz will "Bible" first in XOWA ?

thx, Mac

**Comment by gnosygnu** [comment]
Hi,

Thanks for the detail. Regarding your questions:

> Import was successful, interlinks works. But there was 2 attentions at the end of import

You can ignore these. Wikidata doesn't need page_props (hidden_pages), and there really isn't a category system.

> Please, what mean Page ranks?
> When I'm searching for example Bible in XOWA, first is "United Bible Societies". When i use internet, first "Bible". So when i set wiki-date-pagelinks.sql.gz will "Bible" first in XOWA ?

Page rank is based on Google Page Rank. The url bar uses it to find highly-ranked pages. In your example, "Bible" would come first (or very close to it)

Unfortunately, it can take a day to generate page rank. You also need to generate it through the command-line (the import tool will not generate it).

* If you want a working version with page rank, the easiest way is to download the HTML version of the wiki from Download Central. 
* If you want to build it yourself, you should look at http://xowa.org/home/wiki/Dev/Command-line/Dumps

Hope this helps.

Thanks.

**Comment by MacSquirrel-Jedi** [comment]
thanks for info, helps a lot .) i think, that i can live without Page Rank :) I use primarily Download Central, but i was in tab "Import online" and thought, that there is some setting for import, because of settings "circle" or how to call it :) i made click and after that, didn't found how to abort the proces :)

![bez nazvu](https://user-images.githubusercontent.com/18243759/44477569-55595800-a63b-11e8-8206-e61dea46bed4.jpg)

have a nice day

**Comment by gnosygnu** [comment]
Yeah, the "Import online" page will generate a working version of Wikidata, but it will also pop up those error message above. As I said, you can ignore these error messages.

Let me know if you run into other issues. Thanks.
